src/main/java/com/EighthLight/app/Config.java

```diff
@@ -71,7 +72,7 @@ private void setPlayerOrder() {
             ui.display(Constants.INVALID_CHOICE_PROMPT);
             userInput = ui.getInput();
         }
-        if (userInput.equals("2")) {
+        if (userInput.equals(Constants.COMPUTER_GOES_FIRST)) {
```

Comment:
Updated "2" to constant. Makes users selection more clear. The method still relies on the hash table with null values in order to re-prompt the user if an invalid choice is made. This could be removed, but re-prompting would no longer be available. The method would reverse players on userInput being "2", and would exit on all other inputs.

```java
if (userInput.equals(Constants.COMPUTER_GOES_FIRST)) {
   Collections.reverse(players)
} else {
  // do nothing
}
```